.github/workflows/version_check.yml

```diff
@@ -0,0 +1,198 @@
+name: Check Turing.jl version consistency
```

Comment:
Could this file have a more extensive comment at the top, explaining what the action does?